authentication/src/main/java/io/camunda/authentication/CamundaOidcUserService.java

```diff
@@ -89,4 +95,14 @@ public OidcUser loadUser(final OidcUserRequest userRequest) throws OAuth2Authent
                 .map(GroupEntity::name)
                 .toList()));
   }
+
+  private String getUsernameFromClaims(final Map<String, Object> claims) {
+    return Optional.ofNullable(claims.get(usernameClaim))
+        .map(Object::toString)
```

Comment:
:question: Do we expect/allow anything besides `String` here?

Reply:
My opinion, to keep things simple is that we treat this as we only support alphanumeric values